### PR TITLE
misc: suggest uploading issue repro

### PR DESCRIPTION
**Pull request by connorjclark**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/Bug_report.md [hunk 1]
```diff
@@ -10,6 +10,8 @@ about: Report something working incorrectly
 
 #### Provide the steps to reproduce
 1. Run LH on <affected url>
+  
+<!-- If your page is only local, or is liable to change, consider uploading a repro to https://surge.sh/ so that we can more easily debug the problem. -->
```
Comment (patrickhulce) [inline .github/ISSUE_TEMPLATE/Bug_report.md]:
```suggestion
<!-- If your page is only local, or is liable to change, consider uploading a repro to https://surge.sh/ or creating one at https://jsbin.com/ so that we can more easily debug the problem. -->
```

**Comment by brendankenny** [comment]
Endorsing a particular product feels off to me, but we do use surge in at least one link in our docs. But what about glitch or others?

At the least maybe reword so it's more of "here's the goal, and here's some suggested service(s) that can accomplish the goal"

**Comment by connorjclark** [comment]
@brendankenny 	ok. although I am unfamiliar with glitch, it doesn't seem like a fast tool for uploading code snippets, so I left it out.
